ast2vec/__main__.py

```diff
@@ -65,9 +65,10 @@ def get_parser() -> argparse.ArgumentParser:
 
     bblfsh_args = argparse.ArgumentParser(add_help=False)
     bblfsh_args.add_argument(
-        "--bblfsh", help="Babelfish server's endpoint, e.g. 0.0.0.0:9432.", dest="bblfsh_endpoint")
+        "--bblfsh", default=None, dest="bblfsh_endpoint",
+        help="Babelfish server's endpoint, e.g. 0.0.0.0:9432.")
     bblfsh_args.add_argument(
-        "--timeout", type=int, default=Repo2Base.DEFAULT_BBLFSH_TIMEOUT,
```

Comment:
This shows the hint in `--help`, please leave it

Reply:
ok, I find how to fix it and will create another PR for __main__ to show default value.